mobile-app/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
this entire file is for typescript. If we don't want to use that, we should probably get rid of it.